### PR TITLE
update to idweb 2.16.0

### PR DESCRIPTION
**Pull request by kellyyangsong**

### CODE REVIEW

**Review by brentschmaltz — CHANGES_REQUESTED** [review]
Have a define somewhere so changes can be made in one place.

File: 1-WebApp-OIDC/1-1-MyOrg/WebApp-OpenIDConnect-DotNet.csproj [hunk 1]
```diff
@@ -24,13 +24,13 @@
...
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
-    <PackageReference Include="Microsoft.Identity.Web" Version="2.15.4" />
-    <PackageReference Include="Microsoft.Identity.Web.Certificate" Version="2.15.4" />
-    <PackageReference Include="Microsoft.Identity.Web.Certificateless" Version="2.15.4" />
-    <PackageReference Include="Microsoft.Identity.Web.Diagnostics" Version="2.15.4" />
-    <PackageReference Include="Microsoft.Identity.Web.TokenAcquisition" Version="2.15.4" />
-    <PackageReference Include="Microsoft.Identity.Web.TokenCache" Version="2.15.4" />
-    <PackageReference Include="Microsoft.Identity.Web.UI" Version="2.15.4" />
+    <PackageReference Include="Microsoft.Identity.Web" Version="2.16.0" />
```
[the remark this answers is not in the document]
Comment (kellyyangsong) [reply 1-WebApp-OIDC/1-1-MyOrg/WebApp-OpenIDConnect-DotNet.csproj]: Good idea - done :)

**Comment by jennyf19** [comment]
> Have a define somewhere so changes can be made in one place.

would rather not do that, as this is a sample for customers and we have readmes per sample. So having things stand-alone in the csproj is easier for customers to consume and understand.

**Review by jennyf19 — CHANGES_REQUESTED** [review]
as this is a sample, please revert back to previous thanks.

**Review by jmprieur — APPROVED** [review]
LGTM
Thanks @kellyyangsong

**Pull request merged**